dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScope.kt

```diff
@@ -541,6 +541,14 @@ internal open class RumViewScope(
         writer.write(viewEvent)
     }
 
+    private fun resolveViewDuration(event: RumRawEvent): Long {
+        val duration = event.eventTime.nanoTime - startedNanos
+        if (duration <= 0) {
+            devLogger.w(NEGATIVE_DURATION_WARNING_MESSAGE.format(Locale.US, name))
```

Comment:
@xgouchet , @0xnm do you think we should add a console Log also in RN ? It will be more much more intrusive there as we will have to persist the start time for each view in a `Map<Key, Long>`.

Reply:
The question is if it is actionable on the RN side. Let's maybe try to find the root cause and if we fail to do so, then we may think about adding it there as well considering it will be more visible for the customer (but also more complex to add at the same time).